docs/design/server_side_charts.md

```diff
@@ -0,0 +1,44 @@
+# Supporting charts on the server side
+As discussed in #367, the server side components manager and resourcifier have not yet been refactored, as of this writing, to work with [the new chart format](docs/design/chart_format.md). Presumably, expandybird should not be affected.
+
+## Context
+In discussions about the refactoring on the Slack channel, a number of potential requirements were discussed that could complicate matters, such as:
+* supporting multiple templating technologies within a single chart.
+* supporting multiple templating technologies within a single dependency graph.
```

Comment:
DM supports references between properties in templates of the kind defined by CF and Terraform. Unfortunately, the term `reference` is overloaded to also mean `dependencies` or more precisely `importsin the documentation.